src/coreclr/gc/gcconfig.h

```diff
@@ -144,6 +144,9 @@ class GCConfigStringHolder
     INT_CONFIG   (GCDynamicAdaptationMode,   "GCDynamicAdaptationMode",   "System.GC.DynamicAdaptationMode",   1,                  "Enable the GC to dynamically adapt to application sizes.")                               \
     INT_CONFIG   (GCDTargetTCP,              "GCDTargetTCP",              "System.GC.DTargetTCP",              0,                  "Specifies the target tcp for DATAS")                                                     \
     INT_CONFIG   (GCDBGCRatio,               "GCDBGCRatio",               NULL,                                0,                  "Specifies the ratio of BGC to NGC2 for HC change")                                       \
+    INT_CONFIG   (GCDGen0GrowthPercent,      "GCDGen0GrowthPercent",      "System.GC.DGen0GrowthPercent",      0,                  "Specifies the percentage of the default growth factor")                                  \
+    INT_CONFIG   (GCDGen0GrowthMinFactor,    "GCDGen0GrowthMinFactor",    "System.GC.DGen0GrowthMinFactor",    0,                  "Specifies the minimum growth factor in permil")                                          \
```

Comment:
These will show up in the GC.ConfigurationVariables API - is there a plan to set the value of these like the other configurations?

Reply:
not sure what you mean by "set the value of these"? did you mean set them by something other than env var/runtimeconfig json?

Reply:
Ah, I see you are calling `SetGCDGen0GrowthPercent` and `SetGCDGen0GrowthMinFactor` in the GCConfig API appropriately. However, I think since you added `System.GC.Gen0MaxBudget` in the public key of the configuration, the value that'll show up in the GC Configuration API won't be the right one used by the runtime. We should call SetGCGen0MaxBudget at the right spot since we are now exposing this publicly since we are setting a public key i.e., at https://github.com/dotnet/runtime/blob/7ba3876865539638563d4564d74f02a94e2641d1/src/coreclr/gc/gc.cpp#L44252, right?

Reply:
ahh, yes, of course, thanks for catching that! I pushed a commit to fix it.